src/System.Net.NetworkInformation/tests/FunctionalTests/IPInterfacePropertiesTest.cs

```diff
@@ -19,6 +19,7 @@ public IPInterfacePropertiesTest()
         }
 
         [Fact]
+        [PlatformSpecific(PlatformID.Windows)] // Linux and OSX do not support some of these
```

Comment:
For those it doesn't support, can we add Unix-specific tests that invoke the property and verify it throws PlatformNotSupportedException?